packages/core/src/asciidoctor-core-api.js

```diff
@@ -3047,3 +3047,387 @@ Opal.Asciidoctor.SyntaxHighlighterBase = SyntaxHighlighterBase
 SyntaxHighlighterBase.prototype.registerFor = function (names) {
   SyntaxHighlighter['$register'](this, names)
 }
+
+// Table API
+
+/**
+ * Methods for managing AsciiDoc tables.
+ * @namespace
+ * @extends AbstractBlock
+ */
+var Table = Opal.Asciidoctor.Table
+
+/**
+ * Create a new Table element.
+ * @param {AbstractBlock} parent
+ * @param {Object|undefined} attributes
+ * @returns {Table}
+ */
+Table.create = function (parent, attributes) {
+  return this.$new(parent, toHash(attributes))
+}
+
+/**
+ * Get the caption of the table.
+ * @returns {string}
+ * @memberof Table
+ */
+Table.prototype.getCaption = function () {
+  return this.caption
+}
+
+/**
+ * Get the rows of this table.
+ * @returns {Object} An object with the members "head", "body" and "foot"
+ * @memberof Table
+ */
+Table.prototype.getRows = function () {
+  return this.rows
+}
+
+/**
+ * Get the columns of this table.
+ * @returns {Array<Column>}
+ * @memberof Table
+ */
+Table.prototype.getColumns = function () {
+  return this.columns
+}
+
+/**
+ * Get the head rows of this table.
+ * @returns {Array<Array<Cell>>} - an Array of Array of Cell
+ * @memberof Table
+ */
+Table.prototype.getHead = function () {
+  return this.rows.head
+}
+
+/**
+ * Get the body rows of this table.
+ * @returns {Array<Array<Cell>>} - an Array of Array of Cell
+ * @memberof Table
+ */
+Table.prototype.getBody = function () {
+  return this.rows.body
+}
+
+/**
+ * Get the foot rows of this table.
+ * @returns {Array<Array<Cell>>} - an Array of Array of Cell
+ * @memberof Table
+ */
+Table.prototype.getFoot = function () {
```

Comment:
`getFootRows`

Reply:
getFootRows is definitely better. I'll change it